client/blocks/upsell-nudge/index.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const UpsellNudge = ( { className, ...props } ) => {
```

Comment:
Are these nudged going to be _icon-less_ by default? If so, we'd need to define a default `showIcon` here:

```suggestion
export const UpsellNudge = ( { className, showIcon = false, ...props } ) => {
```

We'd need to also update the readme with that prop.